WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailCoordinator.swift

```diff
@@ -221,16 +229,32 @@ class ReaderDetailCoordinator {
         })
     }
 
-    private func renderPostAndBumpStats() {
+    private func renderPostAndBumpStats(with url: URL? = nil) {
```

Comment:
I was a little bit confused by the `url` as a param here.

I had to check the code to see that `bumpStats` does not rely on this URL, and was wondering why that was set in one of the places that is called but not on the other.

What do you thinnk about instead of having this logic here you just call `self?.scrollToHashIfNeeded(with: url)` in `service.fetchPost`? I guess it will avoid this confusion.

Reply:
This was changed in ee35f5cc93eecd4588b5b6ba05e35a474526e9e0